features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/ImageWireframeHelper.kt

```diff
@@ -33,7 +39,12 @@ internal class ImageWireframeHelper(
         val id = uniqueIdentifierGenerator.resolveChildUniqueIdentifier(view, prefix + index)
 
         @Suppress("ComplexCondition")
-        if (drawable == null || id == null || drawable.intrinsicWidth < 0 || drawable.intrinsicHeight < 0) {
+        if (
+            drawable == null ||
+            id == null ||
+            drawable.intrinsicWidth < 0 ||
```

Comment:
should it be `<=` here?

Reply:
changed to compare to a minimum dimensions constant